packages/shared/src/hooks/analytics/useAnalyticsQueue.ts

```diff
@@ -32,6 +32,13 @@ export default function useAnalyticsQueue({
   const enabledRef = useRef(false);
   const { mutateAsync: sendEvents } = useMutation(
     async (events: AnalyticsEvent[]) => {
+      if (process.env.NODE_ENV === 'development') {
+        // eslint-disable-next-line no-console
+        console.log('useAnalyticsQueue.sendEvents', { events });
```

Comment:
I don't know if logging is the best way here but I wanted to leave some trace in development because event debugging should not be affected.

Reply:
Also thought about replacing `useAnalyticsQueue` hook with "empty" one in `development` but after looking at the rest of the code some of the functions as sendBeacon are required to function on other places.